### PR TITLE
[4.0] Multilang status module

### PR DESCRIPTION
**Pull request by brianteeman**
The notices and warning were displayed in a table. A table should never be used for display purposes and this is not tabular data.

This PR corrects that and puts the notices and warnings as divs

### CODE REVIEW

File: administrator/components/com_languages/tmpl/multilangstatus/default.php [hunk 1]
```diff
@@ -19,117 +19,83 @@
...
-				<td>
-					<span class="fa fa-exclamation-triangle" aria-hidden="true"></span>
-					<span class="sr-only"><?php echo Text::_('WARNING'); ?></span>
-				</td>
-				<td>
-					<?php echo Text::_('COM_LANGUAGES_MULTILANGSTATUS_LANGSWITCHER_UNPUBLISHED'); ?>
-				</td>
-			</tr>
-		<?php endif; ?>
+	<?php if ($this->defaultHome == true) : ?>
```
Comment (Quy) [inline administrator/components/com_languages/tmpl/multilangstatus/default.php]: Indent lines 34-61.

File: administrator/components/com_languages/tmpl/multilangstatus/default.php [hunk 2]
```diff
@@ -19,117 +19,83 @@
...
-				<tr class="table-warning">
-					<td>
-						<span class="fa fa-exclamation-triangle" aria-hidden="true"></span>
-						<span class="sr-only"><?php echo Text::_('WARNING'); ?></span>
-					</td>
-					<td>
-						<?php echo Text::sprintf('COM_LANGUAGES_MULTILANGSTATUS_ERROR_LANGUAGE_TAG', $contentlang->lang_code); ?>
-					</td>
-				</tr>
+			<div class="alert alert-warning">
```
Comment (Quy) [inline administrator/components/com_languages/tmpl/multilangstatus/default.php]: Indent lines 71-74.

File: administrator/components/com_languages/tmpl/multilangstatus/default.php [hunk 3]
```diff
@@ -19,117 +19,83 @@
...
+					<li>
+					<?php echo Text::sprintf('COM_LANGUAGES_MULTILANGSTATUS_CONTACTS_ERROR', $user->name); ?>
+					</li>
+				<?php endforeach; ?>
+				</ul>
+			</div>
 		<?php endif; ?>
-		</tbody>
-	</table>
 	<table class="table table-sm">
```
Comment (Quy) [inline administrator/components/com_languages/tmpl/multilangstatus/default.php]: Indent lines 99-261.

**Comment by brianteeman** [comment]
Drone errors are unrelated to this pr

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on ecb3964f49bf235c7bb0ce6ea968c54d5ad6636d<br /><br />Looks even better than before because of spacing between the divs.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25581">issues.joomla.org/tracker/joomla-cms/25581</a>.</sub>

File: administrator/components/com_languages/tmpl/multilangstatus/default.php [hunk 4]
```diff
@@ -19,279 +19,245 @@
...
-					<?php echo Text::_('COM_LANGUAGES_MULTILANGSTATUS_LANGSWITCHER_UNPUBLISHED'); ?>
-				</td>
-			</tr>
+			<div class="alert alert-warning">
+				<span class="fa fa-exclamation-triangle" aria-hidden="true"></span>
+				<span class="sr-only"><?php echo Text::_('WARNING'); ?></span>
+				<?php echo Text::_('COM_LANGUAGES_MULTILANGSTATUS_LANGUAGEFILTER_DISABLED'); ?>
+			</div>
 		<?php endif; ?>
+	<?php if ($notice_switchers) : ?>
```
Comment (Quy) [inline administrator/components/com_languages/tmpl/multilangstatus/default.php]: Please indent lines 55-61. Then we should be good to go. Thanks.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ac2b7c0bf1d08df9e93cb93ab5035a75d6343a64<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25581">issues.joomla.org/tracker/joomla-cms/25581</a>.</sub>

**Comment by ghost** [comment]
@richard67 can you please retest?

**Comment by Quy** [comment]
Since it is indentation only, a retest is not necessary.

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**